python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -334,21 +365,28 @@ async def write_then_read_reply(
             except AssertionError as e:
                 await self.fault(code=3, report=f"Error writing command {st}: {e}")
                 raise
-            self.writer.write(st.encode() + b"\r\n")
-            await self.writer.drain()
-            try:
-                assert self.reader is not None
-                read_bytes = await asyncio.wait_for(
-                    self.reader.readuntil(b"\r\n"), timeout=_TIMEOUT
+
+            if command not in COMMANDS_DISABLED_FOR_TMA_POINTING:
+                self.writer.write(st.encode() + b"\r\n")
+                await self.writer.drain()
+                try:
+                    assert self.reader is not None
+                    read_bytes = await asyncio.wait_for(
+                        self.reader.readuntil(b"\r\n"), timeout=_TIMEOUT
+                    )
+                except (asyncio.exceptions.TimeoutError, AssertionError) as e:
+                    await self.fault(
+                        code=3, report=f"Error reading reply to command {st}: {e}"
+                    )
+                    raise
+                data: typing.Dict[str, typing.Any] = encoding_tools.decode(
+                    read_bytes.decode()
                 )
-            except (asyncio.exceptions.TimeoutError, AssertionError) as e:
-                await self.fault(
-                    code=3, report=f"Error reading reply to command {st}: {e}"
+            else:
+                data = DATA_FOR_TMA_POINING
+                self.log.debug(
+                    f"WOUTERRRR command {command!r} resulted in reply {data}"
```

Comment:
I'm thinking that perhaps this should be a warning? Though if these commands are sent often then that could easily get annoying and debug is better.

I gently suggest "DISABLED command..."

Reply:
Thanks for catching this. I forgot to remove this log statement since I used it for debugging the unit tests. I removed it now.